packages/salesforcedx-sobjects-faux-generator/test/jest/generator/fauxClassGenerator.test.ts

```diff
@@ -0,0 +1,132 @@
+    }).toThrowError(`SObject category cannot be used to generate metadata ${SObjectCategory.ALL}`);
+  });
+
+  describe('commentToString()', () => {
+
+    it('Should return empty string for empty input', () => {
+      const empty = '';
+      const actual = FauxClassGenerator.commentToString(empty);
+      expect(actual).toEqual(empty);
+    });
+
+    it('Should parse a simple field inline comment', async () => {
+      let firstComment = `/* Please add a unique name${EOL}`;
+      firstComment += `    */${EOL}`;
+      let expectedFirstComment = `${INDENT}/*  Please add a unique name${EOL}`;
+      expectedFirstComment += `    ${EOL}${EOL}${INDENT}*/${EOL}`;
+      const parseFirstComment = FauxClassGenerator.commentToString(firstComment);
+      expect(expectedFirstComment).toEqual(parseFirstComment);
+    });
+
+    it('Should parse a complex field inline comment', async () => {
+      let secondComment = `/******** More complex **************/${EOL}`;
+      secondComment += `**************this is a test **************${EOL}`;
+      secondComment += `/**************/`;
+      let expectedSecondComment = `${INDENT}/*  More complex ${EOL}`;
+      expectedSecondComment += `**************this is a test **************${EOL}`;
+      expectedSecondComment += `${EOL}${INDENT}*/${EOL}`;
+      const parseSecondComment = FauxClassGenerator.commentToString(
+        secondComment
+      );
+      expect(expectedSecondComment).toEqual(parseSecondComment);
+
+    });
+
+    it('Should parse a comment with spaces.', () => {
+      const thirdComment = 'Bring a sweater and/or jacket';
+      let expectedThirdComment = `${INDENT}/* ${thirdComment}`;
+      expectedThirdComment += `${EOL}${INDENT}*/${EOL}`;
+      const parseThirdComment = FauxClassGenerator.commentToString(thirdComment);
+      expect(expectedThirdComment).toEqual(parseThirdComment);
+    });
+
+  });
+
+  describe('generate()', () => {
+    const fakeSfdxPath = '.totallyFake';
+    const expectedFolderPath = join(String(fakeSfdxPath), TOOLS, SOBJECTS_DIR, fakePath);
+
+    let fakeOutput: any;
+    let resetOutputFolderMock: jest.SpyInstance;
+    let getCustomMock: jest.SpyInstance;
+    let getStandardMock: jest.SpyInstance;
+    let generateFauxClassMock: jest.SpyInstance;
+
+    beforeEach(() => {
+      resetOutputFolderMock = jest.spyOn((FauxClassGenerator.prototype as any), 'resetOutputFolder').mockReturnValue(true);
+      generateFauxClassMock = jest.spyOn(FauxClassGenerator.prototype, 'generateFauxClass');
+      getStandardMock = jest.fn();
+      getCustomMock = jest.fn();
+      fakeOutput = {
+        sfdxPath: fakeSfdxPath,
+        getStandard: getStandardMock,
+        getCustom: getCustomMock
+      };
+    });
+
+    it('Should throw if output folder can not be reset.', () => {
+      resetOutputFolderMock.mockReturnValue(false);
+      const expectedError = `No output folder available ${expectedFolderPath}.  Please create this folder and refresh again`;
```

Comment:
I saw in @CristiCanizales PR she was using the nls module to generate the expected error string.  That seems better but I wonder if we should mock the response from that module for unit tests 🤔